src/README.rst

```diff
@@ -16,6 +16,11 @@ To get started, after installation run the following:
 Change Log
 ==========
 
+Unreleased
+----------
+- Provide option to compress packages on application upload. By default, the newly generated compressed package is deleted after successful upload. (#___)
```

Comment:
>rovide  [](start = 3, length = 7)

If they are already providing compressed package, we shouldn't be returning error. It should be a no-op and return success. (I couldn't put that comment in the description, so adding it here.)

Reply:
If they are already providing a compressed package, then they should not use the compress option. I can make that clearer in the help text and error message. Is that what you're referring to?